e2e/assertions/compareScreenshot.js

```diff
@@ -43,8 +72,16 @@ exports.assertion = function(componentName, fileName) {
       .compareTo(resultPath)
       // .ignoreAntialiasing()
       // .ignoreColors()
-      .onComplete(callback)
-
+      .onComplete(data => {
+        if (
+          Number(data.misMatchPercentage) > 0.01 &&
```

Comment:
I just noticed this and I don't understand it. What is `data.misMatchPercentage`? Is that the difference in the comparisons? If so, it should use the `tolerance` variable and not hard coded to 0.01.